src/Symfony/Component/CssSelector/Exception/ExpressionErrorException.php

```diff
@@ -19,6 +19,6 @@
  *
  * @author Jean-François Simon <jeanfrancois.simon@sensiolabs.com>
  */
-class ExpressionErrorException extends ParseException implements ExceptionInterface
+class ExpressionErrorException extends ParseException
```

Comment:
These declarations is just dead code.

It's two components only which has this pattern, so I assume it's re-factoring artifacts (all cases I met pointing to re-factoring).